client/document/index.jsx

```diff
@@ -156,18 +157,28 @@ class Document extends React.Component {
 					<script
 						type="text/javascript"
 						nonce={ inlineScriptNonce }
+						key={ '__inline_script' }
```

Comment:
Irrelevant [`key`](https://reactjs.org/docs/lists-and-keys.html#keys)? Can we drop it?